src/backend/marsha/core/management/commands/check_harvested.py

```diff
@@ -29,22 +29,22 @@ def generate_expired_date():
 
 
 class Command(BaseCommand):
-    """Check every video in harvested and remove them if they are too old."""
+    """Check every video in ready and remove them if they are too old."""
 
     help = (
-        "Check every video having harvested upload state and remove them if "
+        "Check every video having ready upload state and remove them if "
         "they are too old."
     )
 
     def handle(self, *args, **options):
         """Execute management command."""
         videos = Video.objects.filter(
-            upload_state=HARVESTED,
+            upload_state=READY,
```

Comment:
I think you will remove of the VOD videos doing this. 
The condition is I think:

```
live_state=HARVESTED,
starting_at__lte=generate_expired_date(),
```

Reply:
yes you can add the `upload_state` condition too.